api/src/main/java/com/foro/api/service/TopicReplyService.java

```diff
@@ -0,0 +1,141 @@
+package com.foro.api.service;
+
+import com.foro.api.reply.ReplyDTO;
+import com.foro.api.reply.ReplyRepository;
+import com.foro.api.topic.TopicDTO;
+import com.foro.api.topic.TopicRepository;
+import com.foro.api.topic.TopicStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+@Service
+public class TopicReplyService {
+    @Autowired
+    private final TopicRepository topicRepository;
```

Comment:
The autowired now is redundant because spring will inject the object passed in the constructor on the class , please remove the autowired annotation.